spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  before do
+    @user  = FactoryBot.create(:user)
+    @post = FactoryBot.create(:post)
+    @comment = FactoryBot.create(:comment)
+  end
+
+  describe '#create' do
+    let(:post_test) { FactoryBot.create(:post)}
+    context 'user sign in' do
+      it 'create comment successfully' do
+        sign_in @user
+        post :create, params: { comment: { content: 'content test'}, post_id: post_test.id }, format: :js
+        expect(response).to have_http_status(200)
+        expect(post_test.comments.count).to eq(1)  
```

Comment:
Use `change` to check changing of comments count `https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change#expecting-change`